kanidmd_web_ui/src/views/profile.rs

```diff
@@ -49,7 +49,8 @@ impl TryFrom<Entry> for Profile {
             .cloned()
             .ok_or_else(|| "Missing displayname".to_string())?;
 
-        let groups = entry.attrs.get("memberof").cloned().unwrap_or_default();
+        let mut groups = entry.attrs.get("memberof").cloned().unwrap_or_default();
+        groups.sort();
```

Comment:
You should prefer to use sort_unstable https://doc.rust-lang.org/std/vec/struct.Vec.html#method.sort_unstable

Reply:
Makes sense!